webservices/schemas.py

```diff
@@ -394,7 +394,9 @@ class CandidateHistoryTotalSchema(schemas['CandidateHistorySchema'], schemas['Ca
         'ending_image_number': ma.fields.Str(),
         'pdf_url': ma.fields.Str(),
         'is_amended': ma.fields.Boolean(),
+        'document_description': ma.fields.Str(),
     },
```

Comment:
Yeah sorry, I think I confused myself when looking at this.  What was the field that you wished to expose but wasn't showing up?